ping_one/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# CHANGELOG - ping_one
+
+<!-- towncrier release notes start -->
```

Comment:
Please remove this, the changelogs for these integrations are managed by hand.

Reply:
I have removed the changes

Reply:
@akanshi-crest we should remove the towncrier line, the one that this comment is attached to. The other lines you can add back. These changelogs are not managed by towncrier, we have to manage them by hand.